functional-tests/bdd-tests/CRS_transfer_ownership.feature

```diff
@@ -0,0 +1,14 @@
+﻿Feature: CRS_transfer_ownership
+    As a logged in business user
+    I would like to transfer the Cannabis Retail Store ownership
+
+Scenario: Transfer CRS Ownership
+    Given the CRS application has been approved
+    And I am logged in to the dashboard as a private corporation
+    And I click on the Licences tab
+    And the licence fee has been paid
+    And I click on the Transfer Ownership link
+    And I identify the proposed licensee
+    And I complete the consent and declarations
+    And I submit the transfer
+    Then **outcome to be confirmed**
```

Comment:
this is a neat one. You will need to confirm that the application has been initiated (this is indicated on the licence dashboard). Then you will need to login to the proposed licensee account to complete the application. The proposed licensee account will need to exist so that it is searchable.